src/components/generic/Button/Button.jsx

```diff
@@ -54,6 +54,7 @@ export default function Button({
         tabIndex={tabIndex}
         ref={buttonRef}
         style={{height: size, ...style}}
+        // eslint-disable-next-line react/no-unknown-property
```

Comment:
Why was this needed?

Reply:
While committing was observing error for these lines

Reply:
removed it